server/routes/api/v1/surrenderRouter.js

```diff
@@ -0,0 +1,17 @@
+import express from "express"
+import SurrenderApplication from "../../../models/SurrenderApplication.js"
+
+const surrenderRouter = new express.Router()
+
+surrenderRouter.post("/", async (req, res) => {
+  try {
+    const surrenderApplication = new SurrenderApplication(req.body)
+    await surrenderApplication.save();
```

Comment:
In this router endpoint is where we can also create the adoptable pet to associate with the surrender application